testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -573,32 +584,39 @@ def test_sklearn_affinity_propagation(self, ens1):
                               "clusters: {0} {1}".format(len(cc1), len(cc2))
 
     def test_ClusterCollection_init(self, cc):
-        assert np.all(cc.clusters[0].elements == [0, 1, 2]) and \
-               np.all(cc.clusters[1].elements == [3, 4   ]) and \
-               np.all(cc.clusters[2].elements == [5, 6, 7]) and \
-               cc.clusters[0].centroid == 1 and \
-               cc.clusters[1].centroid == 3 and \
-               cc.clusters[2].centroid == 5, \
-                      "ClusterCollection was not constructed correctly"
+        assert_equal(cc.clusters[0].elements, [0, 1, 2])
+        assert_equal(cc.clusters[1].elements, [3, 4])
+        assert_equal(cc.clusters[2].elements, [5, 6, 7])
+        assert_equal(cc.clusters[1].centroid, 3)
+        assert_equal(cc.clusters[2].centroid, 5),
+        "ClusterCollection was not constructed correctly"
```

Comment:
The message will only triggered when the last test fails. Either we remove it or you create a variable `err_msg = ClusterCollection was not constructed correctly` that is passed to every time `assert_equal` is called here.